sources/api/bootstrap-containers/README.md

```diff
@@ -0,0 +1,61 @@
+# bootstrap-containers
+
+Current version: 0.1.0
+
+## Bootstrap containers
+
+Bootstrap containers are host containers that can be used to setup the host
+before services like the ecs-agent, kubelet and docker start. The [configured](../#configuredtarget)
+target depends on the systemd units created for this type of containers, thus
+blocking the init process until all the bootstrap containers exit.
+
+This type of containers are very similar to normal host containers, they come with
+persistent storage and with optional user data. Unlike normal host containers, bootstrap
+containers can't be threated as `superpowered`. However, these containers run as privileged
+and have access to the underlying root filesystem on `/.bottlerocket/rootfs`.
+
+Bootstrap containers have three different modes:
+
+* `always`: with this setting, the container is executed in all boots.
+* `off`: blocks the execution of the container
+* `once`: with this setting, the container only runs on the first boot. Upon completion,
+the container marks itself as `off`. If you need to execute the containers after the first boot,
+you can mark it as `once` and reboot.
+
+## Examples
+Given a bootstrap container with the following configuration:
+
+```toml
+[settings.bootstrap-containers.bear]
```

Comment:
It looks like you can identify multiple bootstrap containers (negating my other question above). How do you order them - or are they explicitly unordered? That should just be called out in the docs..

Reply:
I updated the documentation to mention that bootstrap containers are not executed in any order :+1: . Let me know if you have further questions.